### ISSUE TITLE
(1.20.1) Breaks all right click interactions by doubling it

### ISSUE THREAD

**Issue by R0gyman** [fzzyhmstrs/kf#7]
for the longest time i had no idea what mod did it but when I place down slabs it will put whole blocks down, cant open doors, cant place water, basically anything involving right-click would force it to click twice in one click
not sure what could cause this if its another mod idk

**Comment by R0gyman** [comment]
i have 517 mods so there is no telling what mod did it, just thought id let u know (:

**Comment by fzzyhmstrs** [comment]
Without a log I also can't know, so thanks for letting me know i guess

**Comment by R0gyman** [comment]
> Without a log I also can't know, so thanks for letting me know i guess

How do I get a log without crashing? I don't want anyone else to deal with this issue.
thanks for the quick reply

**Comment by fzzyhmstrs** [comment]
Here's something I want you to try first. If you take out/disable keybind fix, do you still have the problem?

**Comment by R0gyman** [comment]
> Here's something I want you to try first. If you take out/disable keybind fix, do you still have the problem?

Yeah removing the mod fixed my issue for sure it was this mod that was breaking 

Edit: double checked just in case I reinstalled the mod and the issue came back 
if there is any way for me to give a log lmk, I will give it to you as soon as I can get it

edit2: I can also give a modlist if that helps

**Comment by fzzyhmstrs** [comment]
I will check on my end when I'm at my computer

**Comment by fzzyhmstrs** [comment]
I tried in my dev world and can use a bucket no problem, with keybind fix installed. I think there may be another mod that is causing a conflict when KF is installed

**Comment by R0gyman** [comment]
> I tried in my dev world and can use a bucket no problem, with keybind fix installed. I think there may be another mod that is causing a conflict when KF is installed

yeah I thought so lmk if I can somehow give you logs if its possible

**Comment by fzzyhmstrs** [comment]
you can get logs at any time in your `.minecraft` folder under `logs`. file is called latest.log

**Comment by R0gyman** [comment]
https://media4.giphy.com/media/v1.Y2lkPTc5MGI3NjExajJhODc2eDN4NzFtYTNycHhucXNjZXV6NWt3aWFsaHE5NHRnYjc1YSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/JIM9kJFgTFrWslqBAU/giphy.gif

[2024-10-02-2.log.gz](https://github.com/user-attachments/files/17236447/2024-10-02-2.log.gz)


[modlist.txt](https://github.com/user-attachments/files/17236550/modlist.txt)

hope these help if there is anything i can do lmk

**Comment by R0gyman** [comment]
https://github.com/squeeglii/grapplemod-restitched/issues/20
i think i found it its the grapple mod

**Comment by ozmly** [comment]
tested same thing with 400 plus mods, 6 at a time. took ages
indeed was the keybind fix, and I also have the grapplemod
